srcpkgs/sview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+archs="i686 x86_64"
+wrksrc="sview-20_08"
+hostmakedepends="codeblocks ffmpeg libopenal libavcodec libavdevice libavformat libavutil libswscale libwebp libconfig++ libconfig libXpm pkg-config gtk+ libXrandr"
```

Comment:
I'm pretty sure codeblocks is not needed for build with make. Others are not needed for host, right variants are pulled in by *-devel packages.

Reply:
Codeblocks [mentioned in requirements for original project](https://github.com/gkv311/sview/blob/master/docs/INSTALL.md) so I decided to add it, will try to build without it.

I also didn't know that I can ignore packages pulled by dependencies, so stick to [Zen of Python](https://www.python.org/dev/peps/pep-0020/) "explicit is better than implicit" but will remove them.